### ISSUE TITLE
security: Exposed api key in react hook?

### ISSUE THREAD

**Issue by chrysb** [cartesia-ai/cartesia-js#5]
A react hook is nice, but this would leak the API key to the client, right?

**Comment by kbrgl** [comment]
@chrysb Correct. We do not currently have support for issuing keys on behalf of your users, though this is something we've had on the roadmap. We currently recommend people use edge middleware to proxy requests, which adds minimal latency. For an example of this, see https://github.com/ai-ng/swift.

**Comment by charlotte-zhuang** [comment]
@chrysb super old issue, but we added [access tokens](https://docs.cartesia.ai/get-started/authenticate-your-client-applications) a while back.
